### ISSUE TITLE
[pamac-tray] icon stays unchanged (signals still available updates)

### ISSUE THREAD

**Issue by seniordevel** [manjaro/pamac#458]
After successfull installation of some packages, status of pamac-tray icon stays unchanged (signals still available updates). Clicking on it, pamac-updater starts and tells no new updates available. But the icon still does not change.

Pamac version: 10.6.0
Manjaro

**Comment by Rmano** [comment]
If this happens only with flatpak(s), this can be the same as https://github.com/manjaro/pamac/issues/450

**Comment by seniordevel** [comment]
I will look for details with the next update.

If my issue proves to be the same as #450, I will close my report.

**Comment by megathlon** [comment]
I have the same problem, using pamac version 11.7.2-1.
This morning, hovering with the mouse over the try icon showed 177 available updates. After clicking on the try icon, the package manager showed only 1 available update (which was chromium in this case). I installed that one and now the try icon shows 176 available updates.
I noticed that when I try to update the database in pamac, this results in an error message telling the no information about lib32-gcc-libs-git could be generated. Maybe this database inconsistency is related to the wrong number of updates shown.
